util/buildRelease/license_text_for_comment.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2021 Hewlett Packard Enterprise Development LP
+Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
This one only mentions 2022, but it might be correct for it to do so?  @bradcray - thoughts?

Reply:
I think you're correct Lydia.  I'm not familiar with this file, but checking where it's used `add_license_to_sources.py` and that file's log, it appears that this is intended as a stub that's used for a new file, and so therefore should only have the current year.